package.json

```diff
@@ -33,6 +33,7 @@
     "start": "node lib/server",
     "watch": "babel scripts/watch.js | node",
     "test": "npm run lint && npm run check && npm run test:only",
+    "test:coverage": "nyc --report-dir coverage --reporter=lcov  --reporter=text mocha $npm_package_options_mocha",
```

Comment:
I've used the directory coverage so we keep things similar when moving to jest (jest by default creates reports in `coverage`)